docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
+Download the tar file from the DataStax [Downloads site](https://downloads.datastax.com/#csc). Then follow the xref:install.adoc[install] instructions. The tarball contains the Cassandra Source Connector distribution's NAR file.
 
-=== How do I install Cassandra Data Connector?
+=== How many Cassandra Tables can I track?
 
-Download the tar file from the DataStax [Downloads site](https://downloads.datastax.com/#cdc). Then follow the xref:install.adoc[install] instructions. The tarball contains the Cassandra Data Connector distribution's NAR file.
+As many as your system can handle.  Note that each Cassandra table CDC events must be tracked with a separate Cassandra Source Connector.
 
 == Tasks
 
-=== What task can I perform in the DataStax Cassandra Data Connector Installation Console?
+=== What task can I perform in the DataStax Cassandra Source Connector Installation Console?
 
 From the Installation Console, you can:
 
-* Configure the DataStax Cassandra Data Connector environment
-* View and update the DataStax Cassandra Data Connector version and application status
-* Get the host URL and generated password to launch the DataStax Cassandra Data Connector Admin Console
-* Set up and view Grafana graphs, which display metrics collected by the provided Prometheus Operator
-* Analyze DataStax Cassandra Data Connector to collect information you could provide for Support
-* Sync the license that enables DataStax Cassandra Data Connector software
-* View upstream, midstream, and downstream files used by DataStax Cassandra Data Connector
+* Configure the CSC environment.
+* View and update the CSC version and application status.
+* Retrieve the host URL and generated password to launch the CSC Admin Console
+* Set up and view Grafana graphs, which display metrics collected by the provided Prometheus Operator.
+* Analyze the CSC to collect information you could provide for support functions.
+* Sync the license that enables CSC software.
+* View upstream, midstream, and downstream files used by the CSC.
 
-=== What task can I perform in the DataStax Cassandra Data Connector Admin Console?
+=== What task can I perform in the CSC Admin Console?
 
-From the Admin Console, you can:
+From the CSC Admin Console, you can:
 
-* Add and run Pulsar clients
-* Establish credentials for secure connections
-* Define topics that can be published for streaming apps
-* Set up Pulsar sinks that publish topics and make them available to subscribers, such as for a Cassandra database table
-* Control namespaces used by Pulsar
-* Use the Admin API
+* Add and run Pulsar clients.
+* Establish credentials for secure connections.
+* Define topics that can be published for streaming apps.
+* Set up Pulsar sinks that publish topics and make them available to subscribers, such as Cassandra Tables.
+* Control namespaces used by Pulsar.
+* Use the Admin API.
```

Comment:
Are we going to add these topics?

Reply:
This should be on the list.  I'd like to get the software running so I can see this feature in action.